torch/_export/workflow.py

```diff
@@ -0,0 +1,18 @@
+import dataclasses
+from typing import Callable, Tuple
+
+import torch
+
+@dataclasses.dataclass
+class ExportedProgram:
+    module: torch.fx.GraphModule
+    example_inputs: Tuple[torch.Tensor, ...]
+
```

Comment:
Can we add field for in_spec, out_spec, mutation?

Reply:
I can add in_spec and out_spec here. 

Mutation is a bit controversial so I can add them later.